tests/python/topi/python/test_topi_sparse.py

```diff
@@ -212,14 +210,15 @@ def check_device(device):
 
 
 def test_csrmv():
-    verify_dynamic_csrmv(batch=5, in_dim=7, out_dim=1, use_bias=False)
-    verify_dynamic_csrmv(batch=5, in_dim=7, out_dim=1, use_bias=True)
+    verify_dynamic_csrmv(batch=5, in_dim=7, out_dim=1, dtype="float32", use_bias=False)
```

Comment:
As both are 2 diff op, I would suggest to maintain uniformity in Data type input for both the cases.

Reply:
This tests both dtype and use_bias at the same time. I'm trying to avoid adding too much testing overhead. If necessary, I can split it.

Reply:
I meant to add same datatype test cases for both. Please choose to ignore, it is minor. 👍